jnigen/lib/src/elements/elements.dart

```diff
@@ -287,6 +287,8 @@ class Method implements ClassMember {
   @JsonKey(includeFromJson: false)
   late bool isOverridden;
   @JsonKey(includeFromJson: false)
+  late TypeUsage? asyncReturnType;
```

Comment:
Maybe document how/when this is populated.

Is this 100% reliable if suspend fun this is nonNull and if this is not suspend fun this is null? If not, document what data is in here.